whois-query/src/main/java/net/ripe/db/whois/query/pipeline/WhoisServerChannelInitializer.java

```diff
@@ -44,31 +44,33 @@ public class WhoisServerChannelInitializer extends ChannelInitializer<Channel> {
     private final ConnectionPerIpLimitHandler connectionPerIpLimitHandler;
     private final QueryChannelsRegistry queryChannelsRegistry;
     private final TermsAndConditionsHandler termsAndConditionsHandler;
-    private final WhoisEncoder whoisEncoder;
     private final QueryDecoder queryDecoder;
     private final QueryHandler queryHandler;
     private final ApplicationVersion applicationVersion;
     private final boolean proxyProtocolEnabled;
 
+    private final WhoisEncoder whoisEncoder;
+
+
     @Autowired
     public WhoisServerChannelInitializer(final MaintenanceHandler maintenanceHandler,
                                          final QueryChannelsRegistry queryChannelsRegistry,
                                          final TermsAndConditionsHandler termsAndConditionsHandler,
                                          final QueryDecoder queryDecoder,
-                                         final WhoisEncoder whoisEncoder,
                                          final ConnectionPerIpLimitHandler connectionPerIpLimitHandler,
                                          final QueryHandler queryHandler,
                                          final ApplicationVersion applicationVersion,
+                                         final WhoisEncoder whoisEncoder,
```

Comment:
Why re-order the `WhoisEncoder` instance? It was already injected

Reply:
I did revert all the changes in this class